data-tool/flows/batch_delete_flow.py

```diff
@@ -0,0 +1,439 @@
+import math
+from contextlib import contextmanager
+
+from config import get_named_config
+from prefect import flow, task
+from prefect.futures import wait
+from sqlalchemy import Connection, create_engine, exc, text
+from sqlalchemy.engine import Engine
+
+
+businesses_cnt_query = """
+SELECT COUNT(*) FROM businesses
+"""
+
+identifiers_query = """
+SELECT id, identifier FROM businesses
+-- WHERE legal_type IN ('BC', 'ULC', 'CC')
+LIMIT :batch_size
+"""
```

Comment:
Should change this query to select a subset of businesses to delete.

Reply:
i think the default should probably be pretty locked down and whoever uses it will need to manually update the query if they need something else.

So may lock down to legal_type, `BC', 'ULC', 'CC', `businesses.legal name` contains `IMPORT_TEST` and a few `identifiers` by default.

We can nail down exactly what we want as we start using it to wipe corps in dev/test.  

For firms we could lock down identifiers to start with FM0 but it seems like some corps have BC1 prefix so don't think we can use that.